docs/generated/galasactl_runs_submit_local.md

```diff
@@ -19,6 +19,7 @@ galasactl runs submit local [flags]
       --debugPort uint32       The port to use when the --debug option causes the testcase to connect to a java debugger. The default value used is 2970 which can be overridden by the 'galasactl.jvm.local.launch.debug.port' property in the bootstrap file, which in turn can be overridden by this explicit parameter on the galasactl command.
       --galasaVersion string   the version of galasa you want to use to run your tests. This should match the version of the galasa obr you built your test bundles against. (default "0.31.0")
   -h, --help                   help for local
+      --localMaven string      the url of the remote maven where galasa bundles can be loaded from. Defaults to maven central.
```

Comment:
You need to re-run the ./build-locally.sh, as that re-renders this .md file so you can check it in. the build process on teh server should do this, but we currently do it locally.
And this differs from the readme above ? Default should not be to maven central surely ?